homeworks/dmytro.kondaurov_DmitriyKondaurov/3-async/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>3-async</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+<div class="wrapper">
+    <header class="header">
+        <h1 class="header__site-title">Blog</h1>
+        <div class="header__posts-sort">
+            <select class="header__sort-field" name="sort" id="sort" data-sort>
+                <option value="default">sort</option>
+                <option value="a-z">A &#8594; Z</option>
+                <option value="z-a">Z &#8594; A</option>
+            </select>
+            <input class="header__filter-field" type="text" placeholder="Filter" id="filter" data-filter>
+        </div>
+    </header>
+    <main class="main-content">
+        <div class="main-content__loader" data-main-content-loader>
+            <div class="loader__inner"></div>
+        </div>
+        <ul class="post-list" data-main-box>
+            <template data-post-template>
+                <li class="post-list__item" data-post-item>
+                    <h2 class="post-list__item-title" data-post-title>Title</h2>
+                    <p class="post-list__item-text" data-post-text>Lorem ipsum dolor sit amet.</p>
+                    <div class="trash-bnt" data-post-trash-bnt>
+                        <button class="trash-btn__link" data-post-trash-bnt-link>
+                            <img class="trash-btn__icon" src="Trash-Icon.png" alt="Trash" data-post-trash-bnt-icon
+                                 width=66 height=42>
```

Comment:
Потрібно додавати до атрибутів кавички та до розмірів їх юніти
```suggestion
                                 width="66px" height="42px">
```